src/App/Start/useLocalUrls.tsx

```diff
@@ -3,11 +3,19 @@ import { detectLocalIP } from "../../lib/detectLocalIP.js"
 const localIp = detectLocalIP()
 
 export function useLocalUrls(port?: number) {
-  if (!port) return []
+  const urls: {
+    localhost: string | null
+    ip: string | null
+  } = {
+    localhost: null,
+    ip: null,
+  }
 
-  const urls = [`http://localhost:${port}`]
+  if (!port) return urls
```

Comment:
What happens if there is no port available? What is the user experience going to be like?